rollup.config.js

```diff
@@ -1,13 +1,13 @@
 /* eslint-disable flowtype/require-valid-file-annotation, no-console */
-import nodeResolve from 'rollup-plugin-node-resolve'
```

Comment:
I had to convert this to ES5 because karma doesn't support ES6 `import`s and `export`s and I had problems in the past using `babel-register` inside Karma.